test/integration/default/serverspec/default_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe 'logentries_agent::default' do
+
+  it 'installs package' do
```

Comment:
I belive that serverspec output with a context is more obvious, then direct call.